frontend/src/components/EachUser.tsx

```diff
@@ -0,0 +1,19 @@
+import {Participant} from "../types/Participant";
+import {AppUser} from "../types/AppUser";
+
+export default function EachUser(props: {
+    participants: Participant[]
+    user: AppUser,
+}) {
+    const appUser =
+        props.participants.find(participant => participant.email === props.user.email);
+
+
+    return (
+        <div id="oneUser">
+            <div id="name">{props.user.email}</div>
+            <div id="contacts">{props.user.contacts}</div>
+            <div id="participantName">{appUser ? "registered for the event as " + appUser.name :
+                "is not registered for event"}</div>
+        </div>)
+}
```

Comment:
Mhhh.. Ich verstehe nicht ganz den Sinn einer Funktion, die in einer Gruppe eine Person via email findet, um diese Person dann auszugeben - da man diese Person auch einfach so hätte ausgeben können.. Oder hab ich etwas übersehen? 🤔

Reply:
😄  Ich matche hier Participant und User by email. Es sieht so aus, dass Participant und AppUser die gleiche ding ist, aber die sind 2 verschiedene: user existiert immer, seit sign up, hat Email und Passwordhash, Participant ist nur für  eine Veranstaltung, hat Name, Id, Likes und auch Email für Verbindung mit user. Wenn User 10 mal Speed-friending besucht hat, hat er 10 Participants erstellt.